src/server/scripts/EasternKingdoms/zone_silverpine_forest.cpp

```diff
@@ -1,139 +1,2148 @@
+    { 1363.08f, 1048.15f, 53.22223f, 5.253441f },
+    { 1364.08f, 1050.84f, 53.29163f, 5.253441f },
+    { 1366.69f, 1050.31f, 53.34203f, 5.253441f },
+    { 1357.85f, 1050.12f, 52.99823f, 5.253441f }
+};
+
+Position const PortalPos[3] =
+{
+    { 1358.62f, 1054.72f, 53.1200f, 0.f },
+    { 1393.27f, 1021.20f, 53.2225f, 0.f },
+    { 1404.71f, 1063.73f, 60.5617f, 0.f }
+};
+
+Position const GarroshPos = { 1402.45f, 1061.62f, 60.56173f, 3.926991f };
+
+Position const CromushPos = { 1404.71f, 1063.73f, 60.5617f, 2.827433f };
+
+// Grand Executor Mortuus - 44615
+struct npc_silverpine_grand_executor_mortuus : public ScriptedAI
+{
+    npc_silverpine_grand_executor_mortuus(Creature* creature) : ScriptedAI(creature) { }
+
+    void QuestAccept(Player* player, const Quest* quest) override
+    {
+        if (quest->GetQuestId() == QUEST_THE_WARCHIEF_COMETH)
+        {
+            me->GetAI()->SetGUID(player->GetGUID(), PLAYER_GUID);
+
+            me->GetAI()->DoAction(ACTION_START_ANIM);
+        }
+    }
+
+    void Reset() override
+    {
+        _events.Reset();
+
+        _playerGUID = ObjectGuid::Empty;
+
+        _animPhase = 0;
+
+        spawnedList.clear();
+
+        portalList.clear();
+    }
+
+    void SetGUID(ObjectGuid const& guid, int32 id) override
+    {
+        switch (id)
+        {
+            case PLAYER_GUID:
+            {
+                _playerGUID = guid;
+                break;
+            }
+        }
+    }
+
+    void JustSummoned(Creature* summon) override
+    {
+        spawnedList.push_back(summon->GetGUID());
+
+        switch (summon->GetEntry())
+        {
+            case NPC_GARROSH_HELLSCREAM:
+            {
+                summon->CastSpell(summon, SPELL_SIMPLE_TELEPORT);
+
+                _garroshGUID = summon->GetGUID();
+                break;
+            }
+
+            case NPC_HELLSCREEMS_ELITE:
+            {
+                summon->CastSpell(summon, SPELL_SIMPLE_TELEPORT);
+                break;
+            }
+
+            case NPC_PORTAL_FROM_ORGRIMMAR:
+            {
+                portalList.push_back(summon->GetGUID());
+                break;
+            }
+        }
+    }
+
+    void DoAction(int32 param) override
+    {
+        switch (param)
+        {
+            case ACTION_START_ANIM:
+            {
+                if (!_animPhase)
+                {
+                    if (Creature* sylvanas = me->FindNearestCreature(NPC_LADY_SYLVANAS_WINDRUNNER_44365, 100.0f))
+                    {
+                        if (Creature* agatha = me->FindNearestCreature(NPC_AGATHA_44608, 100.0f))
+                        {
+                            _animPhase = 1;
+
+                            _sylvanasGUID = sylvanas->GetGUID();
+
+                            _mortuusGUID = me->GetGUID();
+
+                            _agathaGUID = agatha->GetGUID();
+
+                            _events.ScheduleEvent(EVENT_START_ANIM, 500ms);
+                        }
```

Comment:
no need for all the gaps

```
                            _animPhase = 1;
                            _sylvanasGUID = sylvanas->GetGUID();
                            _mortuusGUID = me->GetGUID();
                            _agathaGUID = agatha->GetGUID();

                            _events.ScheduleEvent(EVENT_START_ANIM, 500ms);
```

is also okay

Reply:
Fixed.